src/conn/conn_layered.c

```diff
@@ -1091,6 +1091,47 @@ __layered_move_updates(
     return (ret);
 }
 
+/*
+ * __layered_clear_ingest_table --
+ *     After ingest content has been drained to the stable table, clear out the ingest table.
+ */
+static int
+__layered_clear_ingest_table(WT_SESSION_IMPL *session, const char *uri)
+{
+    WT_DECL_RET;
+    WT_SESSION_IMPL *internal_session;
+
+    WT_ASSERT(session, WT_SUFFIX_MATCH(uri, ".wt_ingest"));
+
+    internal_session = NULL;
+
+    WT_RET(
+      __wt_open_internal_session(S2C(session), "layered-truncate", false, 0, 0, &internal_session));
+
+    /*
+     * Truncate needs a running txn. TODO we should probably do something more like the history
+     * store and make this non-transactional -- this happens during step-up, so we know there are no
+     * other transactions running, so it's safe.
+     */
+    WT_ERR(__wt_txn_begin(internal_session, NULL));
+
+    /*
+     * No other transactions are running, we're only doing this fast truncate, and it should become
```

Comment:
fast truncate is misleading. We only do slow truncate here as everything is in-memory.

Reply:
Oops, you're right. Thanks, fixed in `2a154e478`.